api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/EriDSDCohortQueries.java

```diff
@@ -1558,4 +1560,132 @@ public CohortDefinition getBreastfeedingPatientsStartingART() {
             hivMetadata.getContinueRegimen().getConceptId()));
     return cd;
   }*/
+  /**
+   * Number of active patients on ART and are marked in the last PU as I OR C on Ficha Clinica
+   *
+   * @retturn CohortDefinition
+   */
+  public CohortDefinition getNumberOfPatientsOnArtAndAreMArkedInLastPuAsIorConFichaClinica() {
```

Comment:
This cohort definition does not filter patients who are **Non-pregnant**, **Non-Breastfeeding** and **not on TB treatment** as per user story.
It may not return correct output

Reply:
Excluded those from the resultant query